server/routes/profile.js

```diff
@@ -4,10 +4,11 @@ const protect = require('../middleware/auth');
 const {
   editProfile,
   loadProfile,
+  loadProfiles,
 } = require('../controllers/profile');
 
 router.route('/edit').put(protect, editProfile);
-
 router.route('/load').get(protect, loadProfile);
+router.route('/profiles').get(loadProfiles)
```

Comment:
```suggestion
router.route('/sitters').get(loadProfiles)
```
Let's give this route a descriptive name